docker/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM clux/muslrust as cargo-build
+FROM clux/muslrust:nightly-2021-02-01 as cargo-build
```

Why not use the [stable tag](https://hub.docker.com/layers/clux/muslrust/stable/images/sha256-71f83effec7fac19ace2acba2de9a022bc0210db0809ec4ece6b43a72575b8cd?context=explore) for the latest stable Rust release:
```suggestion
FROM clux/muslrust:stable as cargo-build
```

This also makes it so the tests are run on the same compiler version as what is being deployed.

---

Good point. I only saw their nightly builds.